src/ostorlab/runtimes/definitions.py

```diff
@@ -32,27 +44,27 @@ class AgentSettings:
     """Agent instance lists the settings of running instance of an agent."""
 
     key: str
-    version: Optional[str] = None
-    bus_url: Optional[str] = ""
-    bus_exchange_topic: Optional[str] = ""
-    bus_management_url: Optional[str] = ""
-    bus_vhost: Optional[str] = ""
-    args: List[defintions.Arg] = dataclasses.field(default_factory=list)
-    constraints: List[str] = dataclasses.field(default_factory=list)
-    mounts: Optional[List[str]] = dataclasses.field(default_factory=list)
+    version: str | None = None
```

Comment:
We support python 3.9 for the moment. Is this working on that version?

Reply:
no, the python 3.9 workflow is failling, reverting